.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+This template is for bug reports. If you are here for another reason, please see below:
+
+1. To propose a new rule, see http://eslint.org/docs/developer-guide/contributing/new-rules
+2. To request a change, see http://eslint.org/docs/developer-guide/contributing/changes
```

Comment:
Is it maybe worth including new rules and changes in the template, and people can just delete whatever sections don't apply?  Or, even if they don't delete them and leave them blank, it doesn't really hurt anything, and we might have a better chance of getting the info we need.